c++/src/Reader.cc

```diff
@@ -1061,7 +1063,6 @@ namespace orc {
 
     schema = convertType(footer->types(0), *footer);
     schema->assignIds(0);
-    previousRow = (std::numeric_limits<uint64_t>::max)();
```

Comment:
Why? In the code immediately above (around lines 1055-1060) we already set previousRow. 
This line is a left-over from a commit I made months ago when I implemented ReaderImpl::seekToRow() method. I forgot to remove it. I discovered it recently while testing because it conflicts with the logic of startNextStripe() and seekToRow() methods.